consensus/src/runway/mod.rs

```diff
@@ -196,7 +197,7 @@ impl<'a, H: Hasher> fmt::Display for RunwayStatus<'a, H> {
 struct RunwayConfig<H: Hasher, D: Data, FH: FinalizationHandler<D>, MK: MultiKeychain> {
     max_round: Round,
     finalization_handler: FH,
-    backup_units_for_saver: Sender<UncheckedSignedUnit<H, D, MK::Signature>>,
+    backup_items_for_saver: Sender<BackupItem<H, D, MK>>,
```

Comment:
It's `BackupItem` because I wanted the `backup` to just listen on a single receiver, but in fact `BackupItem` can be only internal for `backup` and we can remain sending `UncheckedSignedUnit` in `runway` and have a separate sender in 
`alerter` which sends `AlertData`, and `backup` would just convert these things into `BackupItem`. 

in that case `alerter` communicating with `backup` through `runway` would seem unnecessary to me, as `runway` can just create the channels connecting `alerter` with `backup` directly without changing its API.

Would that resolve your suggestion?